yt_dlp/extractor/twitter.py

```diff
@@ -514,6 +570,102 @@ class TwitterIE(TwitterBaseIE):
         },
         'playlist_count': 4,
         'params': {'skip_download': True},
+    }, {
+        'url': 'https://twitter.com/MesoMax919/status/1575560063510810624',
+        'info_dict': {
+            'id': '1575559336759263233',
+            'display_id': '1575560063510810624',
+            'ext': 'mp4',
+            'title': 'md5:eec26382babd0f7c18f041db8ae1c9c9',
+            'thumbnail': r're:^https?://.*\.jpg',
+            'description': 'md5:95aea692fda36a12081b9629b02daa92',
+            'uploader': 'Max Olson',
+            'uploader_id': 'MesoMax919',
+            'uploader_url': 'https://twitter.com/MesoMax919',
+            'duration': 21.321,
+            'timestamp': 1664477766,
+            'upload_date': '20220929',
+            'comment_count': int,
+            'repost_count': int,
+            'like_count': int,
+            'tags': ['HurricaneIan'],
+            'age_limit': 0,
+        },
+    }, {
+        'url': 'https://twitter.com/i/web/status/1579846363218870272',
+        'info_dict': {
+            'id': '1579846355576659970',
+            'display_id': '1579846363218870272',
+            'ext': 'mp4',
+            'title': 'PokiLewd \U0001f51e - @Rizdraws',
+            'thumbnail': r're:^https?://.*\.jpg',
+            'description': '@Rizdraws https://t.co/oSl56cgKKD',
+            'uploader': 'PokiLewd \U0001f51e',
+            'uploader_id': 'pokilewd',
+            'uploader_url': 'https://twitter.com/pokilewd',
+            'timestamp': 1665499699,
+            'upload_date': '20221011',
+            'comment_count': int,
+            'repost_count': int,
+            'like_count': int,
+            'tags': [],
+            'age_limit': 18,
+        },
+        'expected_warnings': ['404'],
+    }, {
+        'url': 'https://twitter.com/Srirachachau/status/1395079556562706435',
+        'playlist_mincount': 2,
+        'info_dict': {
+            'id': '1395079556562706435',
+            'title': str,
+            'tags': [],
+            'uploader': str,
+            'like_count': int,
+            'upload_date': '20210519',
+            'age_limit': 0,
+            'repost_count': int,
+            'description': 'Here it is! Finished my gothic western cartoon. Pretty proud of it. It\'s got some goofs and lots of splashy over the top violence, something for everyone, hope you like it https://t.co/fOsG5glUnw https://t.co/kbXZrozlY7',
```

Comment:
use md5. Same below

Reply:
[This comment](https://github.com/yt-dlp/yt-dlp/pull/5247#discussion_r996330247) explained why I did not want to do that in the first place. It gives you a better error message when doing GraphQL tests, since you can see that only the urls are missing. I can add a test comment so that others cuold understand easier.

Reply:
ah, I missed the other comment since the convo was marked resolved